### PR TITLE
Bump base bound.

### PR DESCRIPTION
**Pull request by athas**

### CODE REVIEW

File: co-log-core.cabal [hunk 1]
```diff
@@ -41,13 +41,14 @@ tested-with:         GHC == 8.2.2
                      GHC == 9.0.2
                      GHC == 9.2.1
                      GHC == 9.2.2
+                     GHC == 9.4.2
```
Comment (vrom911) [inline co-log-core.cabal:44]: For the full `9.4` support that would be nice to also add  the check on CI. This file would need to be patched for that 🙂 

* https://github.com/co-log/co-log-core/blob/main/.github/workflows/ci.yml

**Review by vrom911 — COMMENTED** [review]
Thanks!

File: co-log-core.cabal (hunk 1 above)
Comment (athas) [reply co-log-core.cabal:44]: Done.

**Comment by alaendle** [comment]
Seems that doctest currently doesn't support ghc 9.4 (https://github.com/sol/doctest/pull/382) - so while the co-log-core library builds fine in conjunctions with 9.4.2 it's test suite doesn't. So I guess we should wait for a new doctest release.

**Comment by alaendle** [comment]
Now we should be good to go - I've updated cabal to v3.8.1 since ghc 9.4 requires that. Think this could be merged.

**Comment by sternenseemann** [comment]
doctest works with 9.4 as of 0.20.1, so that constraint can also be relaxed.

**Comment by sternenseemann** [comment]
Indeed! Can a matching change be done on hackage via a revision (ideally for the base bound, too)?

**Comment by alaendle** [comment]
Superseded by https://github.com/co-log/co-log-core/issues/25

**Pull request closed without merging**